### PR TITLE
Update t1_coronavirus_symptom_tracker.md

### PR DESCRIPTION
**Pull request by The-Kergan**
Updated T1 - symptom checkers now included.

### CODE REVIEW

**Comment by The-Kergan** [comment]
@bnorikane , can you please upload the correct pdf to the repo when you do this - as we discussed this morning, I am unable to do it.
`\Dropbox\COVID-19\1-WORK OUTPUT\WEEK 4\FINAL\T1-Coronavirus_Symptom_Tracker_v5.pdf`

File: docs/tools/t1_coronavirus_symptom_tracker.md [hunk 1]
```diff
@@ -1,205 +1,113 @@
...
+<a href="/T1-Coronavirus_Symptom_Tracker_v5.pdf" target="_blank">
     <img class="downloadtools" src="/download-tools.png" />
 </a>
 
+### Why is the tool helpful to you?
 
-### Rationale for Tool
-
-Knowing if anyone has symptoms of Coronavirus helps you decide what to do next.  The most common symptoms of Coronavirus are recent onset of:
+The following symptoms of Coronavirus helps you decide what to do next. The most common symptoms of Coronavirus are recent onset of:
```
Comment (Nelviticus) [inline docs/tools/t1_coronavirus_symptom_tracker.md:9]: The first sentence doesn't make any sense.

File: docs/tools/t1_coronavirus_symptom_tracker.md [hunk 2]
```diff
@@ -1,205 +1,113 @@
...
+| ----------- | ------ | ------- | ------ |
+| 25 Mar 20   | 98.3 N | 98.6 N  | 98.5 N |
+| 26 Mar 20   | 98.1 N | 98.8 N  | 98.6 N |
+| 27 Mar 20   | 98.4 N | 99.6 N  | 99.3 N |
+| 28 Mar 20   | 98.4 N | 100.4 Y | 98.6 N |
 
 ### Other Causes of Body Temperature Variation
 
-There are other factors that effect body temperature ([Web MD](https://www.webmd.com/first-aid/qa/what-can-make-my-body-temperature-change)). Raised temperature should only be considered a coronavirus symptom if it exceeds 100.0 degrees F / 37.8 degrees C.
+There are other factors that effect body temperature ([Web MD](https://www.webmd.com/first-aid/qa/what-can-make-my-body-temperature-change)).
```
Comment (Nelviticus) [inline docs/tools/t1_coronavirus_symptom_tracker.md:86]: affect, rather than effect

File: docs/tools/t1_coronavirus_symptom_tracker.md (hunk 1 above)
Comment (The-Kergan) [inline docs/tools/t1_coronavirus_symptom_tracker.md:9]:
```suggestion
Knowing if anyone has symptoms of coronavirus helps you decide what to do next. The most common symptoms of Coronavirus are recent onset of:
```

File: docs/tools/t1_coronavirus_symptom_tracker.md (hunk 2 above)
Comment (The-Kergan) [inline docs/tools/t1_coronavirus_symptom_tracker.md:86]:
```suggestion
There are other factors that affect body temperature ([Web MD](https://www.webmd.com/first-aid/qa/what-can-make-my-body-temperature-change)). 
```

**Comment by The-Kergan** [comment]
@Nelviticus thanks for spotting the typos. Changes made for approval.

**Pull request merged**